detection_rules/main.py

```diff
@@ -419,7 +419,7 @@ def test_rules(ctx):
 
     rules_config = ctx.obj['rules_config']
     test_config = rules_config.test_config
-    tests, skipped = test_config.get_test_names(formatted=True)
+    skipped, tests = test_config.get_test_names(formatted=True)
```

Comment:
Was this leftover from testing?

Reply:
I think its a bug introduced in your original PR

Reply:
Can you share some example configs you are running that are inverted - I haven't seen this problem

Reply:
It's basically the default skipping one test. Instead of skipping one test, it skips all test except for the one you want to bypass

Reply:
can you share the config for this scenario, the printed output, and your expected results

Reply:
[test.txt](https://github.com/elastic/detection-rules/files/15238604/test.txt)

Essentially I wanted to skip `- tests.test_all_rules.TestValidRules.test_schema_and_dupes` which makes sense because that test cases fails when running the test with no rules. 

Instead of bypassing the test case and running all others, it only tested the one I wanted to skip.

Reply:
```suggestion
    tests, skipped = test_config.get_test_names(formatted=True)
```

Reply:
Ah so the issue was related to the way tests were being pulled `if self.unit_tests.test_only is not None`. This check essentially defaulted to an empty list since the config key was set without the values. I'll reset this and update this check.